Modules/posixmodule.c

```diff
@@ -2588,12 +2588,43 @@ static int
 fill_time(PyObject *module, PyObject *v, int s_index, int f_index, int ns_index, time_t sec, unsigned long nsec)
 {
     assert(!PyErr_Occurred());
+#define S_TO_NS (1000000000LL)
+    assert(nsec < S_TO_NS);
+
+    if (f_index >= 0) {
+        PyObject *float_s = PyFloat_FromDouble(sec + 1e-9*nsec);
```

Comment:
```suggestion
        PyObject *float_s = PyFloat_FromDouble(sec + 1e-9 * nsec);
```
See [PEP7](https://peps.python.org/pep-0007/#code-lay-out).

Reply:
PEP 7 says:

> Always put spaces around assignment, Boolean and comparison operators. In expressions using a lot of operators, add spaces around the outermost (lowest-priority) operators.

I think this code is compliant as it was, and I just moved it up, but I'm happy to change it.